src/filters/highlight.filter.js

```diff
@@ -0,0 +1,15 @@
+export function highlight (value, filter) {
+  if (value && value !== '') {
+    let index = value.indexOf(filter)
+
+    if (index >= 0) {
+      value = value.substring(0, index) +
+        '<strong class="highlight">' +
```

Comment:
No realy fan about adding HTML in a filter with a specific application CSS class `highlight`. But if we don't have the choice, ok.